### DISCUSSION TITLE
3.0.5 Release questions

### DISCUSSION THREAD

**Discussion by Bluetopia** [microsoft/gctoolkit#328] [Q&A] [answered]
Two questions:
* Is a 3.0.5 release planned for the near future, or are there not enough changes yet to justify one.
* I noticed a commit that was changing the Java source version to 17.  Will a future 3.0.5 release be dependent on a Java 17 JVM, or will Java 11 still be supported?

Thanks!

**Comment by karianna** [comment, marked as answer]
@kcpeppe is working on one more major refactor and then we'll have the next official release.
The source compatible level remains at 11 (for now) - see the maven release compiler property in the pom.xml file

**Comment by AlinaHovakimyan** [comment]
Hi @d3r3kk, do we have a new ETA for the next release? 

Thanks in advance.

**Comment by d3r3kk** [comment]
We are working on getting a full release pipeline in place, which needs to happen internally here at Microsoft. The team is learning some brand new systems and should have something by the end of this month.
